tests/ImageSharp.Tests/Formats/Png/PngDecoderTests.cs

```diff
@@ -20,74 +20,94 @@ public class PngDecoderTests
     {
         private const PixelTypes PixelTypes = Tests.PixelTypes.Rgba32 | Tests.PixelTypes.RgbaVector | Tests.PixelTypes.Argb32;
 
+        // This should be exact but for some reason it fails in some build environments.
+        private static readonly ImageComparer ValidatorComparer = ImageComparer.TolerantPercentage(0.0001F, 26);
```

Comment:
Is it travis? Can't we make it environment specific?

Reply:
Seems to be both CI environments.